### PR TITLE
fix: prepend origin to urls without it

### LINKED ISSUE #1075: Another wild "unable to generate request sample" appears
## Describe the bug

Found an API failing to make a request sample, so lets see if we can figure out the cause in case it effects other folks too. 

## To Reproduce

1. Given this OpenAPI demo: https://elements-demo.stoplight.io/?spec=https://api.apis.guru/v2/specs/britbox.co.uk/3.730.267-ref-1-39-0/openapi.yaml#/operations/getBookmarks
2. See error

## Expected behavior

Request sample should show something useful even if its just a simple GET.

## Additional context

**Screenshots**
image

### PR DESCRIPTION
**Pull request by mpodlasin**
Fixes #1075

### CODE REVIEW

File: packages/elements/src/__fixtures__/services/without-origin.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { IHttpService } from '@stoplight/types';
+
+export const httpServiceWithoutOrigin: IHttpService = {
+  id: '?http-service?',
+  name: 'Petstore (OpenAPI v3)',
```
Comment (philsturgeon) [inline packages/elements/src/__fixtures__/services/without-origin.ts:5]: Can we keep petstore usage to a minimum? It's a nit of a running joke in the OAS community, and... not a good joke.

Comment (mpodlasin) [reply packages/elements/src/__fixtures__/services/without-origin.ts:5]: Honestly I just copied some other fixture and modified it for the specific test.

Do you want me to change it?

**Comment by philsturgeon** [comment]
If you could quickly change it, let’s have a “no new pet stores” policy. :)

--
Phil Sturgeon
Product @ Stoplight.io

File: packages/elements/src/utils/http-spec/IServer.ts [hunk 2]
```diff
@@ -3,6 +3,10 @@ import { IServer } from '@stoplight/types';
 export const getServerUrlWithDefaultValues = (server: IServer): string => {
   let url = server.url;
 
+  if (url[0] === '/' && typeof window !== 'undefined') {
```
Comment (mallachari) [inline packages/elements/src/utils/http-spec/IServer.ts:6]: We may also want to do that when there is no `/`. In such case it also shows nothing in `API Base URL` part and breaks RequestSamples. Unless we want that to happen for these kind of inputs.

File: packages/elements/src/__fixtures__/services/without-origin.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      url: 'https://api.int.stoplight.io',
+      name: 'Integration API',
+    },
+    {
+      url: 'https://localhost:4060',
+      name: 'Development API',
+    },
+  ],
+  security: [],
+  securitySchemes: [
```
Comment (mallachari) [inline packages/elements/src/__fixtures__/services/without-origin.ts:38]: Those security schemes seem unnecessary here

**Review by mallachari — COMMENTED** [review]
Works fine with the case described in issue. It doesn't when we remove `/` - see the comment. Unless that's expected behavior, if so then I'll accept :)

**Comment by mpodlasin** [comment]
@mallachari @philsturgeon fixed your comments, please take a second look :)

**Review by philsturgeon — APPROVED** [review]

**Review by mallachari — APPROVED** [review]
:+1:

**Pull request merged**